test/unit/workDiv/CMakeLists.txt

```diff
@@ -6,10 +6,13 @@
 set(_TARGET_NAME "workDivTest")
 
 append_recursive_files_add_to_src_group("src/" "src/" "cpp" _FILES_SOURCE)
+append_recursive_files_add_to_src_group("src/" "src/" "hpp" _FILES_HEADER)
 
 alpaka_add_executable(
     ${_TARGET_NAME}
-    ${_FILES_SOURCE})
+    ${_FILES_SOURCE}
+    ${_FILES_HEADER})
+
 target_link_libraries(
```

Comment:
Additionally 

```
target_include_directories(${_TARGET_NAME} PRIVATE "src")
```

is required to provide the target with the include directory where the hpps are located.

Reply:
Yes, sure. Actually,  I noticed a different usage in  `[alpaka/test/unit/math/CMakeLists.txt](https://github.com/alpaka-group/alpaka/blob/develop/test/unit/math/CMakeLists.txt)` and used the same approach actually.